platform/vechain/api.go

```diff
@@ -195,6 +203,13 @@ func (p *Platform) getTxsByAddress(address string) ([]blockatlas.Tx, error) {
 	return txs, nil
 }
 
+func formatHexToAddress(hex string) string {
```

Comment:
did you consider adding this to `common` package as `add0x` method or something similar?